src/components/buttons/Button/Button.tsx

```diff
@@ -34,11 +34,12 @@ export const Button = (props: IProps) => {
 				className,
 				'Button',
 			)}
-			color={setIntentColor(props, ButtonPropColor.gray)}
+			color={setIntentColor(props, ButtonPropColor.green)}
 			id={id}
-			form={setForm(props, ButtonPropForm.outline)}
-			padding={ButtonPropPadding.m}
-			style={style}
+			secondary
+			form={setForm(props, ButtonPropForm.fill)}
+			padding={ButtonPropPadding.s}
+			style={{ lineHeight: 1.5, ...style }}
```

Comment:
Curious, why not apply this `lineHeight` style to the 'Button' class instead of passing it as part of the style object?

Reply:
`Button` is a global style, I try to avoid adding any more global styles. The `Button` and `PrimaryButton` components don't have CSS files since they're just basically configurations of `ButtonBase`, and I don't want to put it in ButtonBase, because then `TextButtons` will get it - it's really these two buttons that will have consistent font sizes.